src/mip/coefficient_strengthening.jl

```diff
@@ -0,0 +1,178 @@
+"""
```

Comment:
The location of this docstring is a bit odd--usually it would go immediately before (no extra newline) before a function it's describing. Here, it's above `maximal_activity`, but that's just really a helper function.